multiversx_sdk/delegation/delegation_transactions_outcome_parser_types.py

```diff
@@ -6,3 +6,23 @@
 @dataclass
 class CreateNewDelegationContractOutcome:
     contract_address: Address
+
+
+@dataclass
+class ClaimRewardsOutcome:
+    amount: int
+
+
+@dataclass
+class DelegateOutcome:
+    amount: int
+
+
+@dataclass
+class ReDelegateRewardsOutcome:
+    amount: int
+
+
+@dataclass
+class UnDelegateOutcome:
```

Comment:
I know these are the official names in the Protocol, but maybe should we adjust the casing here, in the SDKs? E.g. `Redelegate...`, `Undelegate...`?

Reply:
Renamed